community/py-tempita/APKBUILD

```diff
@@ -3,23 +3,21 @@
 pkgname=py-tempita
 _pkgname=Tempita
 pkgver=0.5.2
-pkgrel=1
+pkgrel=2
 pkgdesc="A small templating language"
-url="http://pythonpaste.org/tempita/"
+url="https://pypi.python.org/pypi/Tempita"
 arch="noarch"
 license="MIT"
-depends=""
-depends_dev=""
-makedepends="python2-dev py-setuptools"
-install=""
-subpackages="py2-${pkgname/py-/}:_py2"
+makedepends="python2-dev py-setuptools python3-dev"
+subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
 source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
-
 builddir="$srcdir"/$_pkgname-$pkgver
 
 build() {
 	cd "$builddir"
 	python2 setup.py build || return 1
+	2to3 -w .
+	python3 setup.py build || return 1	
```

Comment:
`setup.py build` installs files into the same location when run with python2 and python3. The result is that your py2 package contains files converted by 2to3.